scripts/common.py

```diff
@@ -287,15 +286,15 @@ def est_parallel_link_jobs():
     return int(mem_total / mem_per_job)
 
 
-def invoke(cmd, *arguments):
+def invoke(cmd: Command, *arguments: Union[str, List[str]]) -> Tuple[int, str, str]:
     return _invoke(True, cmd, *arguments)
 
 
-def invoke_quietly(cmd, *arguments):
+def invoke_quietly(cmd: Command, *arguments: Union[str, List[str]]) -> Tuple[int, str, str]:
     return _invoke(False, cmd, *arguments)
 
 
-def _invoke(console_output, cmd, *arguments):
+def _invoke(console_output: bool, cmd: Command, *arguments: Union[str, List[str]]) -> Tuple[int, str, str]:
```

Comment:
Can we extract something like this (not sure if these are the best names)?

```python
Args = Union[str, List[str]]
Invocation = Tuple[int, str, str]
```

Reply:
Sure, but it's not worth it for now.